src/container/BooksList.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <table>
+    <tr>
+      <td>BookId</td>
+      <td>Title</td>
+      <td>Category</td>
+      <Book bookId={bookId} title={title} category={category} />
```

Comment:
- [ ] It would be better to loop through the list of books and display each book. This requirement was pointed out earlier by the previous code reviewers. Now that we have access to the `books` array from the `mapStateToProps` we can display the list of books